src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformEditView.tsx

```diff
@@ -381,36 +529,61 @@ export default function TerraformEditView(
 
                   {showModal && (
                     <Dialog
-                      onClose={() => setShowModal(false)}
-                      enforceFocus={false}
-                      className={cx(Classes.DIALOG, 'padded-dialog')}
-                      {...modalProps}
-                      title={getString('pipelineSteps.configFiles')}
+                      {...DIALOG_PROPS}
                       isCloseButtonShown
-                      style={{ padding: '24px' }}
+                      onClose={() => {
+                        setConnectorView(false)
+                        setShowModal(false)
+                      }}
+                      className={cx(css.modal, Classes.DIALOG)}
                     >
-                      <ConfigForm
-                        onClick={data => {
-                          const configObject = {
-                            ...data.spec?.configuration?.spec?.configFiles
-                          }
-
-                          if (configObject?.store.spec.gitFetchType === 'Branch') {
-                            delete configObject.store.spec.commitId
-                          } else if (configObject?.store.spec.gitFetchType === 'Commit') {
-                            delete configObject.store.spec.branch
-                          }
-                          const valObj = cloneDeep(formik.values)
-                          set(valObj, 'spec.configuration.spec.configFiles', { ...configObject })
+                      <div className={css.createTfWizard}>
+                        <StepWizard title={getTitle()} className={css.configWizard} onStepChange={onStepChange}>
+                          <TerraformConfigStepOne
+                            name={getString('cd.configFileStepOne')}
+                            data={values}
+                            isReadonly={readonly}
+                            isEditMode={isEditMode}
+                            allowableTypes={allowableTypes}
+                            setConnectorView={setConnectorView}
+                            selectedConnector={selectedConnector}
+                            setSelectedConnector={setSelectedConnector}
+                          />
+                          {connectorView ? getNewConnectorSteps() : null}
+                          <TerraformConfigStepTwo
+                            name={getString('cd.configFileDetails')}
+                            isReadonly={readonly}
+                            allowableTypes={allowableTypes}
+                            onSubmitCallBack={(data: any, prevStepData: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232256)
```suggestion
                              onSubmitCallBack={(data: unknown, prevStepData: any) => {
```